Change run instructions to run each input json three times (#65)
OpenFreeEnergy/IndustryBenchmarks2024: docs/source/public/overview.rst

Reviewer: > You can find additional information and examples on how to run simulations of the entire network in the "Running the simulations" section of our `CLI tutorial

Do you think we should change this to be a bit clearer that we're doing this a slightly atypical manner? Maybe something like "this is a slightly modified approach to our CLI Tutorial, allowing for execution of a single task per HPC job"?

     relpath=${file:14}  # strip off "network_setup/"
     dirpath=${relpath%.*}  # strip off final ".json"
     jobpath="network_setup/${dirpath}.job"
     for repeat in {0..2}; do
       cmd="openfe quickrun $file -o results_{repeat}/$relpath -d results_{repeat}/$dirpath"
       echo -e "#!/usr/bin/env bash\n${cmd}" > $jobpath
       sbatch $jobpath

>      done 

Reviewer: We should say somewhere that we are happy to help folks adapt this script to their own internal needs as necessary.

>      relpath=${file:14}  # strip off "network_setup/"

Reviewer 2: I think the line above needs to be change to this: 

```bash
 for file in network_setup/transformations/*.json; do
```

not sure if the rest of the rel-paths work then?

Reviewer: I don't have a strong view here - possibly putting thing sin a "transformations" folder might be useful so the top level directory is cleaner. That being said, I consider this to be reasonably low priority so I'm leaving it up to you @hannahbaumann

Author: Ok! I can change it, just to be more similar to the regular CLI output, so that people could reuse the same run script if they used the regular CLI to create their networks.

Author: Edit: I just realized that I would also have to change the run script to create the extra directory and I'm not sure if it's worth it since besides the .jsons files there's only the ligand_network.graphml file in the folder. So maybe we do leave it as is, unless others think it would be beneficial to be consistent @IAlibay @RiesBen

Reviewer: If it means not adding any last minute work, I am more than happy to go with the current state of things @hannahbaumann !

Reviewer 2: Hi, I think it would be great if our folder structure follows this here: 
https://docs.openfree.energy/en/stable/tutorials/rbfe_cli_tutorial.html

in my understanding the final folder structure, after full execution should look like this:

some_benchmark_folder

--network_setup
-----ligand_*json
-----transformations
-------*jsons

--results
----fun1
------shared_*
--------*nc
-----fun1.json
----fun2
------shared_*
--------*nc
-----fun2.json
---- ...

--final_results.tsv

Reviewer 2: will transfer to an issue

     dirpath=${relpath%.*}  # strip off final ".json"
     jobpath="network_setup/${dirpath}.job"
     for repeat in {0..2}; do
       cmd="openfe quickrun $file -o results_{repeat}/$relpath -d results_{repeat}/$dirpath"
       echo -e "#!/usr/bin/env bash\n${cmd}" > $jobpath
       sbatch $jobpath
     done 
   done

> Please reach out to us if you have any questions on how to adapt this script to your internal needs and we'd be happy to assist you in that.

Reviewer: ```suggestion
Please reach out to the openfe team if you have any questions on how to adapt this script to your internal needs, we would be happy to assist with this.
```
Slightly less colloquial :)
